src/components/navBar/navBar.scss

```diff
@@ -0,0 +1,71 @@
+/** Matches "md-tabs md-tabs-wrapper" style. */
+$nav-bar-height: 48px;
+$gms-color-grey-50: #fafafa;
+$gms-color-divider: rgba(0, 0, 0, 0.12);
+$gms-color-secondary-text: rgba(0, 0, 0, 0.54);
+
+.md-nav-bar {
+  background-color: $gms-color-grey-50;
+  border-color: $gms-color-divider;
+  border-style: solid;
+  border-width: 0 0 1px;
+  height: $nav-bar-height;
+  position: relative;
+
+  ul {
```

Comment:
Instead of nesting, introduce individual class that apply to each type of item, e.g.,:
```
.md-nav-bar {
 ...
}

.md-nav-bar-list {
  ...
}

.md-nav-bar-listitem {
  ...
}
```

This keeps the specificity down and can, in some cases, improve performance.